packages/clay-css/src/scss/variables/_tables.scss

```diff
@@ -325,7 +325,7 @@ $c-table: map-deep-merge(
 		autofit-col: (
 			justify-content: center,
 			padding-left: nth($table-cell-padding, 2),
```

Comment:
We should change this one to 0.125rem as well.

Reply:
Hey @pat270 , I was discussing it with @ethib137 and found best to add `0.5rem` on each padding.

Reply:
I will add this to cadmin. Thanks!